### ISSUE TITLE
`solhint >= 3.4.0` linter is broken

### ISSUE THREAD

**Issue by hbarcelos** [dense-analysis/ale#4499]
## Information

**VIM version**

```
VIM - Vi IMproved 9.0 (2022 Jun 28, compiled Mar 21 2023 20:51:42)
Included fixes: 1-1420
```

Operating System:
```
Linux 5.15.104-2-MANJARO #1 SMP PREEMPT Thu Mar 23 01:40:42 UTC 2023 GNU/Linux
```

## What went wrong

`solhint` does not show any errors/warnings.

## Reproducing the bug

Simply open a Solidity file containing issues.

### :ALEInfo
<details>
  <summary>Expand</summary>

  ```
  Command History:
(executable check - success) WORKING_DIR/node_modules/.bin/solhint
(finished - exit code 1) ['/bin/zsh', '-c', 'cd ''WORKING_DIR'' && ''WORKING_DIR/node_modu
les/.bin/solhint'' --formatter compact ''WORKING_DIR/src/Counter.sol'' < ''/tmp/v3svT5T/5/Counter.sol''']
<<<OUTPUT STARTS>>>
There was a problem loading formatter option: compact
Error: Cannot find module './lib/formatters/compact'
Require stack:
- WORKING_DIR/node_modules/solhint/solhint.js
<<<OUTPUT ENDS>>>

  ```
</details>

---

I created a PR to fix the issue: #4494

**Comment by w0rp** [comment]
@hbarcelos I've fixed this now with a patch co-crediting you. Cheers! :beers:
